cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+import dateutil
+
+
+def advance_appliance_date_by_day(appliance):
+    txt_date = appliance.ssh_client.run_command('date --rfc-3339=ns').output
+    appliance_date = dateutil.parser.parse(txt_date)
+    td = datetime.timedelta(days=1)
+    advanced_txt_date = (appliance_date + td).strftime("%Y-%m-%d %H:%M:%S%z")
+    appliance.ssh_client.run_command("date -s '{}'".format(advanced_txt_date))
```

Comment:
Should not we undo what we've caused there? I did not run these tests locally. However it seems to me that all the tests that will run _after_ these two will leave a wrong timestamp in the appliance logs.

Reply:
@jaryn Mine Intension is reducing the complexity as you are trying to increase timedelta by `1 day` in python. Its simple to fire single command in state of it using a shell. I would like to suggest you try this If fails let me know.

Reply:
@jaryn did you try @digitronik request here? Would this work for you?

Reply:
@jaryn Since this is moving back to WIPTEST because of the parameterization comment, I would suggest trying @digitronik suggestion. It is more readable and simpler.

Reply:
@jan-zmeskal To answer your initial question here, I don't think it is necessary to reset the time here. This test is being run with a temp appliance.